DDS: Sanity: Add steps for Share Webhook Configuration in Readme (#20943)
DataDog/integrations-core: sanity/README.md

**Note**: These steps enable collection of project activity logs.

### Retrieve the Datadog Webhook URL

1. Select an existing API key or create a new one by clicking one of the buttons below: <!-- UI Component to be added by Datadog team -->

> **Note**: To set up the webhook, please follow the instructions provided in the Manual Webhook Configuration. Alternatively, you can use the Share Webhook Configurations option to apply a pre-configured setup.

Reviewer: ```suggestion
**Note**: To set up the webhook, please follow the instructions provided in the Manual Webhook Configuration. Alternatively, you can use the *Share Webhook Configurations* option to apply a pre-configured setup.
```

> ### To set up the webhook, please follow the instructions provided in the manual Webhook Configuration. Alternatively, you can use the Share Webhook Configurations option to quickly apply a pre-configured setup.

Reviewer 2: ```suggestion
**Note**: To set up the webhook, please follow the instructions provided in the manual Webhook Configuration. Alternatively, you can use the Share Webhook Configurations option to apply a pre-configured setup.
```
This would be better in a Note rather than a heading.
Is there a link to the manual Webhook configuration? If so we should link to that here.

Author: I’ll update the heading to `Note`. There are no links available for the manual webhook configurations; these steps were documented based on the live setup configuration.

      }
    ```
    **Note**: It is recommended to _**uncheck**_ the **Trigger webhook when drafts are modified** checkbox in the Drafts section.
9. Ensure **POST** is selected under **HTTP method** in the **Advanced settings section**.
10. Click **Save**.

### Share Webhook Configuration

> **Note:** These are alternative steps for **Manual Webhook Configuration**. Please follow only one set of instructions.

Reviewer 2: ```suggestion
**Note**: These are alternative steps for **Manual Webhook Configuration**. Follow only one set of instructions.
```

Author: Done

1. Log in to your [Sanity account][2] as an Administrator.

> 2. To configure the webhook automatically, please follow this [link][6].

Reviewer 2: ```suggestion
2. To configure the webhook automatically, please follow the [Sanity webhooks documentation][6].
```

Author: This link is not the Sanity webhooks documentation; rather, it allows users to automatically fill all the required configurations in Sanity using this link, so they don’t have to fill all the fields manually. I’ll update the name to `Sanity webhook configuration`. Please let me know what you think!

3. Update the existing URL with the Datadog webhook URL.
4. Click **Apply webhook**.
5. In the configuration dialog, set the following parameters:

>     - **Organization:** Select the organization from the dropdown list.

Reviewer 2: ```suggestion
    - **Organization**: Select the organization from the dropdown list.
```

Author: Done

>     - **Project:**: Select the relevant project from the dropdown list.

Author: Done

>     - **Dataset:** Select `* (all datasets)`.

Reviewer 2: ```suggestion
    - **Dataset**: Select `* (all datasets)`.
```

[3]: https://www.sanity.io/manage
[4]: https://www.sanity.io/docs/webhooks#
[5]: https://docs.datadoghq.com/help/

> [6]: https://www.sanity.io/manage/webhooks/share?name=sanity-datadog&description=&url=https%3A%2F%2Fhttp-intake.logs.datadoghq.com%2Fapi%2Fv2%2Flogs%3Fdd-api-key%3D%3CYourDatadogAPIKey%3E%26ddsource%3Dsanity%26service%3Dactivity-logs&on=create&on=update&on=delete&filter=&projection=%7B%0A%20%20%20%20%22documentId%22%3A%20_id%2C%20%0A%20%20%20%20%22documentType%22%3A%20_type%2C%20%0A%20%20%20%20%22projectId%22%3A%20sanity%3A%3AprojectId()%2C%0A%20%20%20%20%22datasetName%22%3A%20sanity%3A%3Adataset()%2C%0A%20%20%20%20%22action%22%3A%20%22document.%22%20%2B%20delta%3A%3Aoperation()%2C%0A%20%20%20%20%22beforeValues%22%3A%20before()%2C%0A%20%20%20%20%22afterValues%22%3A%20after()%2C%0A%20%20%20%20%22timestamp%22%3A%20now()%0A%7D&httpMethod=POST&apiVersion=v2025-02-19&includeDrafts=&includeAllVersions=&headers=%7B%7D

Author: This is an additional request from Sanity to generate a share webhook config URL. Since this is an auto-generated configuration URL, we’re unable to manually modify or remove any part of it.
